### DISCUSSION TITLE
Issue with search results list display

### DISCUSSION THREAD

**Discussion by spiderb4by** [microsoft-search/pnp-modern-search#3555] [Q&A] [answered]
First, I'm a bit new to this. I apologize in advance if my question seems rather silly.

I'm working with a Search Results webpart that is returning the ''Title'' and some other custom properties from items in a list. To give you some context, my custom columns are based on data from the term store and have already been mapped to RefinableStrings.

The problem I'm having is that when I try to show the results using the ''Detailed List'' (not fully sure about the translation) layout, Pnp returns the whole object and not just its ''Title'' value. It looks something like this:

<img width="236" alt="2024-02-21 16_15_50-ABC Cascades - Accueil et 3 pages de plus - Travail – Microsoft​ Edge" src="https://github.com/microsoft-search/pnp-modern-search/assets/160788261/cd158309-3103-4e18-ada1-fa9f38331421">

Now, I was able to truncate a bit of that horror by using the getSummary expression like this, in the layout options: 
{{getSummary (slot item @root.slots.RawMaterial)}}

<img width="730" alt="gerer_les_colonnes" src="https://github.com/microsoft-search/pnp-modern-search/assets/160788261/345f77eb-6faf-4a7c-89e1-fac88d3b3357">

The problem is, I still get the unecessary characters ''GP0|#;L0|#0 etc''. 

<img width="360" alt="result" src="https://github.com/microsoft-search/pnp-modern-search/assets/160788261/09c13ac0-b949-4737-adc6-019c6c934c6a">

Is there any way I can get only the title?

I would really appreciate your help!

**Comment by kasperbolarsen** [comment, marked as answer]
To me it looks a lot like you have mapped the ows_taxid_[propertyName] to the RefinableString? If you just need to see the "label" of the Term value you can map to ows_[propertyname] in stead

Try to use one of the tools here, https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/ as that will show you the values in each of the Managed Properties :-)

**Reply by spiderb4by** [reply]
It works when I use the AutoRefinableString instead of the RefinableString as the data source.

Thank you! :)

<img width="614" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/160788261/9ab1e2bb-7285-436e-b041-7f15a30b2484">

<img width="361" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/160788261/779ec767-6489-43e6-a358-92174a8d5ec5">
